awscli/topics/config-vars.rst

```diff
@@ -0,0 +1,209 @@
+:title: AWS CLI Configuration Variables
+:description: Configuration Variables for the AWS CLI
+:category: General
+:related command: configure, configure get, configure set
+
```

Comment:
You can add ``:related topic: s3-config`` to get this topic to link to the s3 configuration topic when a related items section is added for topics.